README.md

```diff
@@ -1,36 +1,78 @@
-Piper
+Piper [![CircleCI](https://circleci.com/gh/libratbag/piper.svg?style=svg&circle-token=7082ad7a7fea706fff66f1547649dca32e446cb0)](https://circleci.com/gh/libratbag/piper)
 =====
 
-GTK application to configure gaming mice. Piper uses libratbag via ratbagd.
+Piper is a GTK+ application to configure gaming mice, using libratbag via
+ratbagd.
+
+Piper is currently being redesigned. For the mockups, see the [Redesign
+Wiki][wiki-redesign].
 
 Dependencies
 ============
 
-Piper uses the ratbagd Python bindings. It doesn't require ratbagd itself to
-be running, but well, it'll be a little useless without ratbagd.
+* python3
+* gobject-introspection
+* gtk3
+* python-gobject
+* python-cairo
+* python-lxml
+* libratbag
 
-Sources
-=======
+In order to run Piper, `ratbagd` has to be running. To see how, see [its
+README][ratbagd-readme].
+
+Installing Piper
+================
+
+Piper uses the [meson build system][meson] which in turn uses [ninja][ninja] to
+build and install itself. Run the following commands to clone Piper and
+initialize the build:
+
+```
+$ git clone https://github.com/libratbag/piper.git
+$ cd piper
+$ meson builddir --prefix=/usr/
+```
+
+To build or re-build after code-changes, run:
 
-https://github.com/libratbag/piper
+```
+$ ninja -C builddir
+$ sudo ninja -C builddir install
+```
 
-https://github.com/libratbag/ratbagd
-https://github.com/libratbag/libratbag
+Note: `builddir` is the build output directory and can be changed to any other
+directory name.
 
 Contributing
 ============
 
-Yes please. As you can see, years of design experience have gone into the
-user interface and it is clearly the pinnacle of what is possible.
-Feel free to send patches or pull requests or submit improvements for
-the general design of the app.
+Yes please. Note, however, that Piper is currently being redesigned as a Google
+Summer of Code project and as such, things will change significantly. As such,
+it's best to contact us first to see what you could do. Otherwise, hang back and
+the work might be done for you! Current status can be seen [here][kanban], the
+design mockups can be seen on the [Wiki][wiki-redesign].
```

Comment:
I am not sure about this one too (@whot?).

My concerns are that yes, we should let know that there is an ongoing redesign progress, but writing this down in the README seems overkill, especially when we might forget to change this in the future.

How about a simple header on the github repository page (which I just edited)?